protobuf/syft_proto/messaging/v1/message.proto

```diff
@@ -17,17 +17,19 @@ message SyftMessage {
     // IsNoneMessage contents_is_none_msg = 4;
     ObjectMessage contents_object_msg = 5;
     // ObjectRequestMessage contents_object_request_msg = 6;
-    CommandMessage contents_command_msg = 7;
-    // PlanCommandMessage contents_plan_command_msg = 8;
-    // SearchMessage contents_search_msg = 9;
+    TensorCommandMessage contents_tensor_cmd_msg = 7;
+    // PlanCommandMessage contents_plan_cmd_msg = 8;
+    // WorkerCommandMessage contents_worker_cmd_msg = 9;
```

Comment:
I don't really get why these (are some others) are commented out?

Reply:
They don't have Protobuf schemas yet. The commented out lines are essentially a TODO list.